composer.json

```diff
@@ -4,7 +4,7 @@
     "keywords": [ "file", "attachment", "laravel", "upload" ],
     "homepage": "https://github.com/czim",
     "require": {
-        "php": "^7.4|^8.0",
+        "php": ">=8.0.0",
```

Comment:
I would prefer `"^8.0"`, bit cleaner notation, does the same and does not assume it'll work in PHP 9 ;]

Reply:
Makes sense, changed.